src/mpu9250_interface.cpp

```diff
@@ -20,67 +22,108 @@ void MPU9250Interface::init() {
     i2c.write(MPUREG_I2C_SLV0_REG, data, 1);          // I2C slave 0 register address from where to begin data transfe
     i2c.write(MPUREG_I2C_SLV0_CTRL, readData, 1);     // Read 6 bytes from the magnetometer
 
+    // TODO
```

Comment:
Tip: be consistant in comment style. The general standard is the use of ```///<``` or the use of docblocks: ```/** **/```